### DISCUSSION TITLE
Homekit Secure Video

### DISCUSSION THREAD

**Discussion by adekka** [samemory/homebridge-eufy-security-control#324] [Q&A] [answered]
I saw the development of secure video and it's added to the [beta](https://github.com/homebridge/homebridge/tree/beta-1.4.0). Great work!

Was wondering how or when to test the functionalities with this plugin. I installed the 1.4.0-beta.4 of homebridge and the v1.1.1-beta.14 of this plugin. This combination does not sow HSV features.

**Comment by schliemann** [comment, marked as answer]
As I read the current homebridge support for hksv it is not available yet.

https://github.com/homebridge/HAP-NodeJS/pull/920

**Comment by adekka** [comment]
Just checking, i saw the updates up nodeJS and homebridge. These  include HSV. Should this plugin also make an update to make HSV work?

**Reply by schliemann** [reply]
It requires changes to the plugin. You are welcome to create an issue.
